<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MicroscannerTest</class>
 <widget class="QMainWindow" name="MicroscannerTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>915</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Open Sans</family>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">Microscanner Test</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="capturelbl">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>0</y>
      <width>260</width>
      <height>851</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Camera Setting</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QPushButton" name="openCam_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>261</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Cam</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>260</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Exposure</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <widget class="QComboBox" name="exposureMode">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>241</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="exposure_slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <number>40</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>260</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>White Balance</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <widget class="QComboBox" name="WBMode">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>241</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="R_slider">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>70</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="G_slider">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>100</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="B_slider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>R</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>G</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>340</y>
       <width>260</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Brightness</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <widget class="QSlider" name="brightness_slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="digitalShift_slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>DigitalShift</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>460</y>
       <width>260</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>GainRaw</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <widget class="QSlider" name="gainRaw_slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="gamma_slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Gamma</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>600</y>
       <width>260</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Denoising</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <widget class="QSlider" name="denoising_slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="denoising_lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Value</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="denoising_cb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>730</y>
       <width>260</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Sharpeness</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <widget class="QSlider" name="sharpness_slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="denoising_slider_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Value</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="sharpness_cb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="AreaScan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>260</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Area Scan</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QLineEdit" name="widthScan_text">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="heightScan_text">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="setArea_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>241</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Area Scan</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Lebar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Panjang</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="AreaScan_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>260</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Koordinat Awal Slide</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QLineEdit" name="xHomeSlide_text">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>211</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="yHomeSlide_text">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>211</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="setHomeSlide_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>241</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Koordinat Awal</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>21</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>21</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>260</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Setup Serial</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QTextEdit" name="status_txt">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>261</width>
       <height>60</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="comCombo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>261</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="connect_btn">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="comRefresh_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fokus_btn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Fokus</string>
     </property>
    </widget>
    <widget class="QPushButton" name="prepare_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Persiapan</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="time_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>770</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="total_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>770</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>260</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Nama Slide</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QLineEdit" name="namaSlide_text">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>261</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="namaSlide_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>241</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Nama Slide</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_10">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>640</y>
      <width>381</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Posisi</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QLabel" name="currRow_lbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Baris</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="currCol_lbl">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Kolom</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="currImg_lbl">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>140</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>140</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gambar</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_11">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>640</y>
      <width>381</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Total</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QLabel" name="sumRow_lbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Baris</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="sumCol_lbl">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Kolom</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="sumImg_lbl">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>140</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>140</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gambar</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="start_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>261</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Capture">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>720</y>
      <width>261</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
